src/shared/components/InputBase/InputBase.styles.ts

```diff
@@ -27,6 +27,11 @@ export const FormGroup = styled.label<FormGroupProps>`
     font-size: ${typography.sizes.body1};
     line-height: 28px;
 
+    input,
+    textarea {
+      color: ${colors.gray[50]};
+    }
+
```

Comment:
Text in inputs is black now. 
![image](https://user-images.githubusercontent.com/51168865/134481018-07ac62b2-1c87-4292-90e2-661937822e33.png)
you should move this block before 
```  
  input,
  button,
  textarea {
...
```

Reply:
Crap, I did it because of stylelint.